dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/UnpackagedTests/MrtCoreUnpackagedTests.csproj

```diff
@@ -84,17 +84,18 @@
   <!-- The powershell command below updates the TestAdaptersPaths path in the runsettings file, in order to allow the test to run in VS Test Explorer. -->
   <PropertyGroup>
     <PostBuildEvent>copy /Y "$(OutputPath)..\Microsoft.ApplicationModel.Resources\Microsoft.Internal.FrameworkUdk.dll"  "$(OutputPath)Microsoft.Internal.FrameworkUdk.dll"
+    copy /Y "$(OutputPath)..\MRM\MRM.dll"  "$(OutputPath)MRM.dll"
```

Comment:
>copy /Y "$(OutputPath)..\MRM\MRM.dll"  "$(OutputPath)MRM.dll" [](http://example.com/codeflow?start=4&length=61)

is this required? I see it in the output folder.

Reply:
> > copy /Y "$(OutputPath)..\MRM\MRM.dll"  "$(OutputPath)MRM.dll"
> 
> is this required? I see it in the output folder.

Yes. Those copy commands can probably be replaced with ReferenceCopyLocalPaths like MrtCoreManagedTest but in the spirit of surgically unblocking I've tried to avoid making larger changes. The copy command this way doesn't work for the managed (packaged) test because it only copies into $(OutDir) but MSTest actually creates $(OutDir)AppX with copies from OutDir **of what it thinks belongs**.

You could change MrtCoreUnpackagedTest to use ReferenceCopyLocalPaths but it's worth looking into removing the need for the runsettings file. Perhaps with more love in its [AssemblyInitialize] (copy files, call AddDllDirectory, other). Something to consider in the future